packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -187,6 +185,7 @@ export class OdspDocumentService implements IDocumentService {
                     }
                     throw new Error("Disconnected while uploading summary (attempt to perform flush())");
                 },
+                this.mc.config.getNumber("Fluid.Driver.Odsp.snapshotFormatFetchType"),
```

Comment:
I'd recommend you to explicitly assign values to elements of SnapshotFormatSupportType enum, and not rely on defaults of a compiler.